src/node_modules.cc

```diff
@@ -323,8 +323,8 @@ void BindingData::GetNearestParentPackageJSON(
   // Check if the path has a trailing slash. If so, add it after
   // ToNamespacedPath() as it will be deleted by ToNamespacedPath()
   bool slashCheck = !path_value.ToString().empty() &&
-                    path_value.ToString().back() ==
-                        std::filesystem::path::preferred_separator;
+                    path_value.ToStringView().ends_with(
```

Comment:
The previous line is not needed `path_value.ToString()` creates an unnecessary string.

Can you also remove this in other places?

Reply:
Fixed.